app/views/spree/admin/shared/_new_resource_links.html.haml

```diff
@@ -0,0 +1,4 @@
+.form-buttons.filter-actions.actions
+  = button t(:create), 'icon-ok'
+  %span.or= t(:or)
+  = button_link_to t(:cancel), collection_url, icon: 'icon-remove'
```

Comment:
I wonder why the change from link_to_with_icon to button_link_to. What is the differrence between these two methods? Will the user notice any difference?

Reply:
The only difference is purely aesthetic - whether or not it looks like a button, as far as I could tell, but maybe I am mistaken.